activate.ps1

```diff
@@ -0,0 +1,66 @@
+#
+# This file must be used by invoking ". .\activate.ps1" from the command line.
```

Comment:
Copied from AspNetCore, needed to run the baseline update scripts.

Reply:
Will this still be required after https://github.com/dotnet/extensions/pull/4587? It seems that the changes going there will fix the issues you were hitting, so I would rather not add this script if not required any longer.

Reply:
Ah, that would help. It's a tradeoff though, do you write one script for activation, or do you code it into every other script? It's also useful if you want to call any dotnet commands directly.

Reply:
If you use `start-code.cmd` then you can run `dotnet` directly from the VS Code's terminal as the script bootstraps all the necessary variables.